### PR TITLE
[gcc-4.9.2] fixed git to include EB perl and path (REVIEW)

### PR DESCRIPTION
**Pull request by rjeschmi**
Some of git, especially the submodules code, is written in Perl.

I had some problems with the base version of perl included in centos6, so it makes some sense to use the version of perl available in EB.

### CODE REVIEW

File: easybuild/easyconfigs/g/git/git-2.4.1-GCC-4.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/git/git/archive']
+
+preconfigopts = 'make configure && '
+
+# Work around git build system bug.  If LIBS contains -lpthread, then configure
+# will not append -lpthread to LDFLAGS, but Makefile ignores LIBS.
+configopts = "--with-perl=${EBROOTPERL}/bin/perl --enable-pthreads='-lpthread'"
+
+dependencies = [
```
Comment (boegel) [inline easybuild/easyconfigs/g/git/git-2.4.1-GCC-4.9.2.eb]: style remark: please move this up below the `sources`/source_urls` (also in the other git 2.x easyconfig you're touching in this PR)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/12d5bbf1765685fa3445 for a full test report.

**Comment by ocaisa** [comment]
Still waiting for style update as requested by @boegel otherwise good to go

**Comment by rjeschmi** [comment]
I believe I've resolved the style comments, thanks @ocaisa @boegel

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a1f0411a86d8d8e27bb2 for a full test report.

**Comment by boegel** [comment]
Thanks @rjeschmi, and @ocaisa for reviewing, going in!

**Pull request merged**